db/migrate/20250301135740_remove_redundant_listing_indexes.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveRedundantListingIndexes < ActiveRecord::Migration[7.1]
+  def up
```

Comment:
since we're adding a composite index on [:movie_id, :list_id], the db can use that index and we don't need the individual indexes